tests/chainer_tests/links_tests/connection_tests/test_deconvolution_2d.py

```diff
@@ -150,7 +150,7 @@ def check_backward(self, x_data, y_grad):
             params.append(self.link.b)
 
         gradient_check.check_backward(
-            self.link, x_data, y_grad, params, eps=1e-2)
+            self.link, x_data, y_grad, params, eps=1e-2, rtol=1e-3, atol=1e-4)
```

Comment:
Why not use `dtype='d'`

Reply:
Because this is connection test.

Reply:
>Because this is connection test.

What does this mean?

Reply:
Sorry, Because this is link test.

Reply:
Well, why don't link tests need `dtype`?
In my understanding, underlying numerical computations are not different between link tests and function tests.

Reply:
Sorry, I was misunderstanding. I will fix

Reply:
OK!